content/docs/other-guides/support.md

```diff
@@ -0,0 +1,168 @@
++++
+title = "Support"
+description = "Where to go with questions and suggestions"
+weight = 110
++++
+
+This page describes the Kubeflow resources and support options that you can
+explore when you encounter a problem, have a question, or want to make a
+suggestion about Kubeflow.
+
+TODO: This page currently mentions an "alpha" status whereas the version policy
```

Comment:
Noting the TODO.

@abhi-g and @jlewi I'm interested in your thoughts here. Our v1 versioning policy page ([preview](https://deploy-preview-1308--competent-brattain-de2d6d.netlify.com/docs/reference/version-policy/#kubeflow-application-matrix)) uses **stable**, **beta**, and **experimental** for the application statuses. I'm thinking we should change "experimental" to **alpha**. Some pages/components already use the term "alpha", and it fits better with "beta". What do you think?